### ISSUE TITLE
aircrack-ng - huge memory consumption (leak?) with hex wep dictionary

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1138]
**Reported by varchar on 7 Jun 2013 09:57 UTC**

Aircrack-ng 1.2 beta1 with following command:
```
aircrack-ng -a 1 -n 64 -w h:wep64hex some.cap 
```
and with content of wep64hex dict like this: 
```
...
06:37:80:25:61
06:37:80:25:66
06:37:80:27:16
06:37:80:27:19
06:37:80:27:61 
...
```
eat GiB's of RAM after few minutes of run, and then must/will be killed.
----
Here not the right place to ask, but how to use stdin as hex dict?
``` -w h:- ```
does not work for me, unlike
``` -w - ```
Thanx and sorry.

**Comment by aircrack-ng** [comment]
**Comment by varchar on 7 Jun 2013 09:57 UTC**

Found this as duplicate of [#685](#685)

**Comment by aircrack-ng** [comment]
**Modified by misterx on 7 Jun 2013 09:57 UTC**

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 7 Jun 2013 09:57 UTC**

It will be fixed or no? 685 (with patch attached) 4 years old!

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 7 Jun 2013 09:57 UTC**

Have you tested the patch? It doesn't fix the issue.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 7 Jun 2013 09:57 UTC**

Fixed in aircrack-ng/aircrack-ng@a954db50290631c96d5f65af8d8fa31c9c35f7af.
